src/m365/spo/commands/group/group-user-list.ts

```diff
@@ -0,0 +1,93 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { GroupUserInfo } from './GroupUserInfo';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  groupId?: number;
+  name?: string;
+}
+
+class SpoGroupUserListCommand extends SpoCommand {
+
+  public get name(): string {
+    return commands.GROUP_USER_LIST;
+  }
+
+  public get description(): string {
+    return `List the members of a SharePoint Group`;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['Title', 'UserPrincipalName', 'Id', 'Email'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Retrieving the list of members from the SharePoint group :  ${args.options.groupId ? args.options.groupId : args.options.name}`);
+    }
+
+    const requestUrl: string = `${args.options.webUrl}/_api/web/sitegroups/${args.options.groupId
+      ? `GetById('${encodeURIComponent(args.options.groupId)}')`
+      : `GetByName('${encodeURIComponent(args.options.name as string)}')`}/users`;
+
+    const requestOptions: any = {
+      url: requestUrl,
+      headers: {
+        'accept': 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get<GroupUserInfo[]>(requestOptions)
+      .then((groupUserList: GroupUserInfo[]): void => {
+        logger.log(groupUserList);
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '--groupId [groupId]'
+      },
+      {
+        option: '--name [name]'
```

Comment:
We missed it in the spec: we should name this option `groupName` to keep it consistent with `groupId` and also, because the last noun in the command's name is `user` while here we're referring to group's name rather than user's name.